### PR TITLE
LRAC-6455 Apply retry logic to identity events

### PR DESCRIPTION
**Pull request by fcodias**
https://issues.liferay.com/browse/LRAC-6455

### CODE REVIEW

**Comment by fcodias** [comment]
ci:test:sf

**Comment by fcodias** [comment]
cc @Masterfoni

File: modules/apps/analytics/analytics-client-js/src/client.js [hunk 1]
```diff
@@ -89,21 +115,156 @@ class Client {
...
+
+		if (this.delay !== this.initialDelay) {
+			this.delay = this.initialDelay;
+			this._startsFlushLoop();
+		}
+	}
+
+	/**
+	 * Go through the queue and send the messages.
+	 * If a queue fail the processing loop isdelayed
```
Comment (andreldm) [inline modules/apps/analytics/analytics-client-js/src/client.js:169]: typo

File: modules/apps/analytics/analytics-client-js/src/client.js [hunk 2]
```diff
@@ -89,21 +115,156 @@ class Client {
...
+	/**
+	 * Increase the interval time and restart processing loop
+	 */
+	onRequestFail() {
+		this.delay = getRetryDelay(++this.attemptNumber, LIMIT_FAILED_ATTEMPTS);
+		this._startsFlushLoop();
+	}
+
+	/**
+	 * Resetinterval time and restart processing loop
```
Comment (andreldm) [inline modules/apps/analytics/analytics-client-js/src/client.js:156]: typo

**Comment by andreldm** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/analytics/analytics-client-js/src/client.js [hunk 3]
```diff
@@ -89,21 +115,156 @@ class Client {
...
+		this.attemptNumber = 1;
+
+		if (this.delay !== this.initialDelay) {
+			this.delay = this.initialDelay;
+			this._startsFlushLoop();
+		}
+	}
+
+	/**
+	 * Go through the queue and send the messages.
```
Comment (andreldm) [inline modules/apps/analytics/analytics-client-js/src/client.js:168]: s/queue/queues?

**Comment by andreldm** [comment]
Pull request submitted to https://github.com/andreldm/liferay-portal/pull/38. [See changes here.](https://github.com/andreldm/liferay-portal/compare/1f05a32c0a8a~1...1f05a32c0a8a) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**